### ISSUE TITLE
[BUG] Opensearch Database Error 400

### ISSUE THREAD

**Issue by fandigunawan** [frikky/Shuffle#684]
**Describe the bug**
Sometimes the application could not save and run workflow into OpenSearch due to invalid datatype.

**To Reproduce**
1. Put node (accidentally on my browser set the location into "integer" type location rather than "float")
2. Save or run
3. Error 400 
![image](https://user-images.githubusercontent.com/10239907/150606348-51c6500f-e959-4083-a86c-695e078cdcd4.png)

**Expected behavior**
Apps executed correctly

**Screenshots**

** Debug logs (NOT APPLICABLE FOR CLOUD)**
I modified shuffle-shared to display OpenSearch Error as follow:
```
shuffle-backend | 2022/01/21 20:35:35 [DEBUG] Index name = workflowexecution, id = 29e1eb71-c9bd-493b-80a4-eccd8323fcef, body = {"type":"workflow","status":"EXECUTING","start":"dc08a25d-75ea-4c90-bcef-9154ae7fe723","execution_argument":"","execution_id":"29e1eb71-c9bd-493b-80a4-eccd8323fcef","execution_org":"2ae418d4-95c1-4579-be09-0f8ada908a9a","started_at":1642797335,"completed_at":0,"workflow_id":"80ca7667-985f-4808-915c-584708275872","last_node":"","authorization":"a67e35d9-fdc1-42bd-8c8d-5f1f9196fb08","result":"","project_id":"shuffle","locations":null,"workflow":{"actions":[{"app_name":"Shuffle Tools","app_version":"1.1.0","description":"","app_id":"841ea71b-d3d5-4b42-bc9a-eea6534542d6","errors":[],"id":"dc08a25d-75ea-4c90-bcef-9154ae7fe723","is_valid":true,"isStartNode":true,"sharing":true,"label":"Change Me","public":true,"generated":false,"environment":"Shuffle","name":"repeat_back_to_me","parameters":[{"description":"","id":"","name":"call","example":"Repeating: Hello World","value":"Hello world","multiline":true,"options":null,"action_field":"","variant":"STATIC_VALUE","required":false,"configuration":false,"tags":null,"schema":{"type":""},"skip_multicheck":false,"value_replace":null,"unique_toggled":false}],"execution_variable":{"description":"","id":"","name":"","value":""},"position":{"x":449.5,"y":446},"authentication_id":"","category":"","reference_url":"","sub_action":false,"source_workflow":"","run_magic_output":false,"run_magic_input":false,"execution_delay":0},{"app_name":"Testing","app_version":"1.0.0","description":"","app_id":"abe58fff-e8d5-48f5-b98a-d04afa8143d0","errors":[],"id":"0a64375f-a84a-4409-ac2e-74d28d5781fb","is_valid":true,"sharing":true,"label":"Testing_1","public":true,"generated":false,"environment":"Shuffle","name":"hello_world","parameters":[],"execution_variable":{"description":"","id":"","name":"","value":""},"position":{"x":637.796195652174,"y":387.22282608695656},"authentication_id":"","category":"Testing","reference_url":"","sub_action":false,"source_workflow":"","run_magic_output":false,"run_magic_input":false,"execution_delay":0}],"branches":[{"destination_id":"0a64375f-a84a-4409-ac2e-74d28d5781fb","id":"9fa2419a-f72a-4919-a044-84025c6b5e04","source_id":"dc08a25d-75ea-4c90-bcef-9154ae7fe723","label":"","has_errors":false,"conditions":null,"decorator":false}],"visual_branches":[],"triggers":[],"schedules":[],"comments":[],"configuration":{"exit_on_error":false,"start_from_top":false,"skip_notifications":false},"created":1642797299,"edited":1642797310,"last_runtime":0,"id":"80ca7667-985f-4808-915c-584708275872","is_valid":true,"name":"test","description":"","start":"dc08a25d-75ea-4c90-bcef-9154ae7fe723","owner":"0b7ac4af-0188-4ea1-8d1c-be0491c4909d","sharing":"private","org":[{"name":"","id":"2ae418d4-95c1-4579-be09-0f8ada908a9a","users":null,"role":"","creator_org":"","image":"","child_orgs":null}],"execution_org":{"name":"","id":"2ae418d4-95c1-4579-be09-0f8ada908a9a","users":null,"role":"","creator_org":"","image":"","child_orgs":null},"org_id":"2ae418d4-95c1-4579-be09-0f8ada908a9a","workflow_variables":null,"execution_environment":"","previously_saved":true,"categories":{"siem":{"name":"","count":0,"id":"","description":""},"communication":{"name":"","count":0,"id":"","description":""},"assets":{"name":"","count":0,"id":"","description":""},"cases":{"name":"","count":0,"id":"","description":""},"network":{"name":"","count":0,"id":"","description":""},"intel":{"name":"","count":1,"id":"","description":""},"edr":{"name":"","count":0,"id":"","description":""},"IAM":{"name":"","count":0,"id":"","description":""},"other":{"name":"","count":0,"id":"","description":""}},"example_argument":"","public":false,"default_return_value":"","contact_info":{"name":"","url":""},"published_id":""},"results":[],"org_id":"","execution_source":"default","execution_parent":"","execution_source_node":"","execution_source_auth":"","sub_execution_count":0,"priority":0}
shuffle-backend | 2022/01/21 20:35:35 [400 Bad Request] {
shuffle-backend |   "error" : {
shuffle-backend |     "root_cause" : [
shuffle-backend |       {
shuffle-backend |         "type" : "illegal_argument_exception",
shuffle-backend |         "reason" : "mapper [workflow.actions.position.y] cannot be changed from type [float] to [long]"
shuffle-backend |       }
shuffle-backend |     ],
shuffle-backend |     "type" : "illegal_argument_exception",
shuffle-backend |     "reason" : "mapper [workflow.actions.position.y] cannot be changed from type [float] to [long]"
shuffle-backend |   },
shuffle-backend |   "status" : 400
shuffle-backend | }
```
The root cause is due to position.x and position.y that is parsed as "long" rather than "float"
In this case the main cause is due to position.y
![image](https://user-images.githubusercontent.com/10239907/150606730-ab3cc9f4-fc7f-4420-b201-8af905715319.png)

**Comment by frikky** [comment]
> ```
> 
> ```

ooh, interesting. I would've thought our system parsed it into a float naturally when we read it. Bug to be fixed, thanks!

**Comment by frikky** [comment]
Fixed in 0.9.65+
